components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -642,11 +642,13 @@ interface JsonRpcService {
   GetNetworkUrl(CoinType coin) => (string network_url);
 
   // Obtains the native balance (e.g. ETH for Ethereum) for the address
-  GetBalance(string address, CoinType coin) => (string balance, ProviderError error, string error_message);
+  GetBalance(string address, CoinType coin, string chain_id) => (string balance, ProviderError error, string error_message);
 
   // Obtains the contract's ERC20 compatible balance for an address
   GetERC20TokenBalance(string contract,
-                       string address) => (string balance, ProviderError error, string error_message);
+                       string address,
+                       CoinType coin,
```

Comment:
This is ETH only I believe, I don't think we need to pass coin type for this API.